frontend/src/components/TodoTask/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from 'antd';
+import { StarFilled, StarOutlined, BorderOutlined, CheckSquareOutlined } from '@ant-design/icons';
+import { blue } from '@ant-design/colors';
+
+export type IconButtonProps = {
+  onClick(): void;
+  color?: string;
+  variant: 'star' | 'check';
+  state: boolean;
+};
+
+export const IconButton = ({ onClick, color = blue.primary, state, variant }: IconButtonProps) => {
```

Comment:
A może?

```suggestion
export const IconChecbox = ({ onClick, color = blue.primary, state, variant }: IconButtonProps) => {
```

Reply:
Jeden to gwiazdka, drugi to checbox, a oba są buttonami, więc uznałem, że button to lepsza wersja